### PR TITLE
convert: add example for creating a gif

### PR DESCRIPTION
**Pull request by ShaunKarran**

### CODE REVIEW

File: pages/common/convert.md [hunk 1]
```diff
@@ -17,3 +17,7 @@
 - Horizontally append images:
 
 `convert {{image1.png}} {{image2.png}} {{image3.png}} +append {{image123.png}}`
+
+- Create gif from series of images:
```
Comment (agnivade) [inline pages/common/convert.md]: Create a gif from a series of images with a given delay in (unit?) between them.

File: pages/common/convert.md [hunk 2]
```diff
@@ -17,3 +17,7 @@
...
+
+`convert {{image1.png}} {{image2.png}} {{image3.png}} -delay 100 {{animation.gif}}`
```
Comment (agnivade) [inline pages/common/convert.md]: 100 in tokens.

File: pages/common/convert.md (hunk 1 above)
Comment (ShaunKarran) [reply pages/common/convert.md]: Thanks for the comments! :)

File: pages/common/convert.md (hunk 2 above)
Comment (ShaunKarran) [reply pages/common/convert.md]: The other examples set a specific value for the purpose of the example. I was following that format.

Comment (agnivade) [reply pages/common/convert.md]: Yes, we prefer specific values in examples. I just want the value to be in tokens.

**Pull request merged**